include/specDefWrapper.h

```diff
@@ -0,0 +1,21 @@
+#pragma once
+
+// this test is to not break pic-build or cmake builds
+#ifndef CMAKE_VERSION
+    #define CMAKE_VERSION=NO_CMAKE_IN_SPEC
+    #define CUPLA_STREAM_ASYNC_ENABLED 0
+    #ifdef SPEC_CUDA
+        #define CUPLA_STREAM_ASYNC_ENABLED 1
```

Comment:
This will throw a warning that `CUPLA_STREAM_ASYNC_ENABLED? is already defined. Best is to be explicit in each accelerator section even if it looks like copy past.